src/xinterpreter.cpp

```diff
@@ -38,6 +38,9 @@ namespace xpyt
 {
     void interpreter::configure_impl()
     {
+        m_release_gil = new py::gil_scoped_release();
```

Comment:
I think this deserves a comment (having a scope object dynamically instantiated is weird at first sight) and a delete in the destructor of the interpreter.

Reply:
Made it an `std::unique_ptr` as discussed in person :)